cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -31,7 +31,9 @@
 
 @pytest.fixture(scope="module")
 def vm_name():
-    return "test_snpsht_" + fauxfactory.gen_alphanumeric()
```

Comment:
**REQUIRED**: If I understand correctly, you just need to change the scope of the fixture to function to get a new vm name every time right?